src/codewhisperer/util/codeWhispererSessionStates.ts

```diff
@@ -3,16 +3,18 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-export class CodeWhispererStates {
-    static #instance: CodeWhispererStates
+const performance = globalThis.performance ?? require('perf_hooks').performance
+
+export class CodeWhispererSessionStates {
```

Comment:
Is "states" actually needed in the name?
```suggestion
export class CodeWhispererSession {
```